1.8/tutorial/redis-package.md

```diff
@@ -0,0 +1,46 @@
+---
+post_title: Install First Package
+nav_title: Installing Package
+menu_order: 2
+---
+
+Welcome to part 2 of the DC/OS 101 Tutorial.
+
+# Prerequisites
+By now, you have a running DC/OS cluster and the DC/OS CLI installed and configured. If that isn't the case, please follow these [instructions](/docs/1.8/tutorial/cli/).
+Furthermore, we use [jq](https://stedolan.github.io/jq/) as a json processor to simplify some of the below commands.
+
+# Objective
+By the end of this session we will have installed our first service--redis--from the DC/OS universe. Redis is a key-value store, which we will use for persisting data throughout the tutorial.
```

Comment:
I capitalized Redis where I am talking about the app, the package is called redis. So where talking about the package I left it lower case.